### DISCUSSION TITLE
[2023 PROJECTS TURN-IN] Final Projects discussions and links

### DISCUSSION THREAD

**Discussion by williamgilpin** [williamgilpin/cphy#21] [archived]
Please post names, groups, topic areas, and eventually links to final projects in this thread.

Once you have your repo name and URL settled, you can go ahead and edit your post to add it here even if it's unfinished (I won't grade it until the deadline in December). If you later change the URL, please edit the original post so that I don't get confused.

**Comment by ehansen99** [comment]
Yuchen Lin, Tong Wang, Daniel Urbanski, Erik Hansen
Plasma: Magnetorotational Instability

**Reply by williamgilpin** [reply]
could you post that paper you were showing me friday? i liked the figures

**Reply by ehansen99** [reply]
Yes, the paper we talked about on Friday is attached.
[Riquelme_2012_ApJ_755_50.pdf](https://github.com/williamgilpin/cphy/files/13045475/Riquelme_2012_ApJ_755_50.pdf)

**Comment by abbyperryman** [comment]
Ravi Koka, Anish Pandya, Patrick Devaney, Abigail Perryman
Wave propagation on quantum graphs

**Comment by ccyehintx** [comment]
Chieh-Chih (George) Yeh, Hanting Chen, Ching-Yu (Jimmy) Huang
Stochastic processing of soft shell hard sphere collisions

**Reply by williamgilpin** [reply]
this sounds awesome! I don't think that I got a chance to chat with your group, but I would be happy to arrange a time to chat sometime

**Comment by bhollo0329** [comment]
Bryce Holloway, Lauren Larson, Josey Wright, Bryan Dinh
Using neural networks to identify jets in the ATLAS detector

**Reply by williamgilpin** [reply]
Thank you! I'll link the OG thread here:
https://github.com/williamgilpin/cphy/discussions/26

**Comment by shoshanareich** [comment]
Sophia Epstein, Noah Lindsell, and Shoshi Reich
Symplectic Adjoint Method and Neural ODEs for the Hodgkin Huxley model
https://github.com/williamgilpin/cphy/discussions/28

**Reply by williamgilpin** [reply]
hi here is the Dynamical Components Analysis NeurIPS paper (i think it is a particularly well-written paper)

https://proceedings.neurips.cc/paper_files/paper/2019/file/704cddc91e28d1a5517518b2f12bc321-Paper.pdf

**Reply by sepstein22** [reply]
Our finalized repo can be found here [computational_brain](https://github.com/sepstein22/computational_brain.git)

Sophia Epstein, Noah Lindsell, Savannah Pinedo, and Shoshi Reich

**Comment by aditi-pujar** [comment]
Asad Hussain, Shivan Mittal, Kody Anderson, Aditi Pujar
Bayesian Inference and Building MCMC Samplers

**Comment by ycaophysics** [comment]
Yuxuan Cao, Timothy Araujo, Hongming Wang
Reconstructing accelerated electron beams using machine learning

**Reply by williamgilpin** [reply]
https://maziarraissi.github.io/PINNs/

**Reply by ycaophysics** [reply]
Here's the gist of our code (with some trials of PINNs):
https://gist.github.com/ycaophysics/fd0ae800c94bc8bce6bf0dd38405864a

**Comment by clobrien-ut** [comment]
Chloe O'Brien and Greg Sehr
Building a generative model for more efficient Lux-Zeplin detector electronics response simulations

**Reply by williamgilpin** [reply]
Here's Ram Reddy's really nice project from last year. He focused on training a discriminator network to detect MadGraph simulations for two different standard models. So if a GAN is a generator and discriminator pair, the generator is a fixed MadGraph module, and the discriminator is a DNN

in your case you'd want to train a generator, though the discriminator idea might be useful if you are trying to differentiate two types of events, etc

https://github.com/ramreddy-physics/Madgraph_Search

**Reply by williamgilpin** [reply]
Also pasting my notes from our chat, so I can find them easily:

+ Event happens at t=0. Some initial photons are picked up by photomultiplier (PM) tubes
+ Electrons are produced in the medium, which undergo a biased random walk towards the PM due to scattering and an applied electric fields
+ Difference in arrival time between initial and later signals gives the z depth of the event
+ Due to random walk dynamics, deeper events will have higher XY localization errors due to scattering
+ Ignore charge loss at wall, other edge effects
+ Range of events: since photons, to tens of thousands at a time

Data: electrical signals at an XY array of PM tubes. Extremely reminiscent of EEG electrode arrays, spike sorting and localization via deconvolution

**Comment by ccyehintx** [comment]
Hi,
This is the link for our group project:
https://github.com/ccyehintx/Soft_Shell_project/tree/main
Chieh-Chih (George) Yeh, Hanting Chen, Ching-Yu (Jimmy) Huang

Thank you

**Comment by ravikoka** [comment]
Patrick Devaney, Ravi Koka, Anish Pandya, Abby Perryman

How to train your electron: Anderson localization on the lattice and random graphs. 

Link: https://github.com/ravikoka/qgraph

**Comment by clobrien-ut** [comment]
Chloe O'Brien and Gregory Sehr
Generative Model of Lux-Zeplin Detector Electronic Response
https://github.com/clobrien-ut/cphy/tree/main/final_project

**Comment by DanUrban000** [comment]
Yuchen Lin, Tong Wang, Daniel Urbanski, and Erik Hansen
Collisionless MRI
https://github.com/ehansen99/Python-Kinetic-Magnetorotational-Instability?tab=readme-ov-file

**Comment by BryanD-UT** [comment]
Bryan Dinh, Lauren Larson, Bryce Halloway, Josie Wright
https://github.com/laurenlarson/comp_phys_2023/tree/main

**Comment by shvnmttl** [comment]
Asad Hussain, Shivan Mittal, Aditi Pujar, Kody Anderson
[Final project repo](https://github.com/Potatoasad/Computational-Physics-Final-Project)
[Final project docs](https://potatoasad.github.io/Computational-Physics-Final-Project/index.html)
[Docs for sampler package](https://potatoasad.github.io/Computational-Physics-Final-Project/quicksampler.samplers.html)
